### PR TITLE
PXP-397: [CLI] Support editing the CHANGELOG & Post to Slack

### PR DESCRIPTION
**Pull request by jk-gan**
## Summary

Pending API

Ticket: [PXP-397](https://mindvalley.atlassian.net/browse/PXP-397)

### Added
- [x] add changelog edit on the cli side
Screenshot 2022-11-15 at 5 38 07 PM
Screenshot 2022-11-15 at 5 37 52 PM

- [x] allow wukong backend to post on slack once the deployment is done

### Fixed
- [x] remove unused dependencies

### CODE REVIEW

**Comment by sauron-droid** [comment]
@onimsha: Unable to find goose. Have you checked the garden?

<details>

In response to [this](https://github.com/mindvalley/wukong-cli/pull/28#issuecomment-1303025764):

>/honk


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by sauron-droid** [comment]
@jk-gan: PR needs rebase.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/28#" title="Author self-approved">jk-gan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by onimsha — CHANGES_REQUESTED** [review]
Also, please use the term `changelog` and `CHANGELOG` everywhere, instead of `changelogs` or `CHANGELOGS`. 

WHY ? Take from this wiki page https://en.wikipedia.org/wiki/Changelog

>Although the canonical naming convention for the file is ChangeLog,[[2]](https://en.wikipedia.org/wiki/Changelog#cite_note-2) it is sometimes alternatively named as CHANGES or HISTORY (NEWS is usually a different file reflecting changes between releases, not between the commits). Another convention is to call it a CHANGELOG.[[1]](https://en.wikipedia.org/wiki/Changelog#cite_note-keepachangelog-1) Some [project maintainers](https://en.wikipedia.org/wiki/Project_maintainer) will append a .txt suffix to the file name if the changelog is [plain text](https://en.wikipedia.org/wiki/Plain_text), a .md suffix if it is in [Markdown](https://en.wikipedia.org/wiki/Markdown), or a .rst suffix if it is in [reStructuredText](https://en.wikipedia.org/wiki/ReStructuredText).

File: src/commands/deployment/execute.rs
Comment (jk-gan) [reply src/commands/deployment/execute.rs:443]: alright

**Review by onimsha — APPROVED** [review]

**Pull request merged**
